### PR TITLE
PIM-5900: Move backtogrid button and enable button to the title div, and fix css

### PR DESCRIPTION
**Pull request by pierallard**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n
| Added Behats                      | n
| Changelog updated                 | n
| Review and 2 GTM                  | TODO
| Migration script                  | n
| Tech Doc                          | n

capture du 2016-08-03 10 36 55

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/views/CategoryTree/form.html.twig [hunk 1]
```diff
@@ -52,13 +52,17 @@
...
             {% endif %}
-        {% elseif parentId %}
-            {{ elements.backLink(path(route ~ '_categorytree_edit', { id: parentId }), 'btn.to parent') }}
         {% endif %}
         {{ elements.submitBtn('', 'ok') }}
     {% endset %}
 
-    {{ elements.page_header(title, buttons, null, null, elements.updated(form.vars.id)) }}
+    {% set titleButtons %}
+        {% if (form.vars.value.id is null) and parentId %}
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Resources/views/CategoryTree/form.html.twig]: why this button is only shown when id is null ?

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 2]
```diff
@@ -1,8 +1,8 @@
 @import 'less/variables';
 @import 'less/mixins';
 
-.navbar-extra .btn {
-  margin-left: 4px;
+.navbar-extra .btn+.btn {
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: `.btn+.btn` ? can you explain it please, I didn't know this  ?

**Comment by skeleton** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/views/CategoryTree/form.html.twig (hunk 1 above)
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/views/CategoryTree/form.html.twig]: I moved the logic from the previous back-to-grid button, and it was a
if (form.vars.value.id)
...
else if parentId
-- it was here ---
endif

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less (hunk 2 above)
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: Explained IRL. For the others: 
http://www.w3schools.com/cssref/sel_element_pluss.asp

**Comment by ClementGautier** [comment]
GTM

**Pull request merged**